Dizzy/Screens/Login/AppInfosView.swift

```diff
@@ -0,0 +1,199 @@
+//
+//  AppInfoView.swift
+//  Dizzy
+//
+//  Created by stas berkman on 05/06/2019.
+//  Copyright © 2019 Dizzy. All rights reserved.
+//
+
+import UIKit
+
+protocol AppInfosViewDelegate: class {
+    func appInfoButtonPressed(appInfosView: AppInfosView, type: AppInfoType)
+}
+
+enum AppInfoType: String {
+    case about = "About"
+    case termsOfUse = "Terms of Use"
+    case privacyPolicy = "Privacy Policy"
+    case contactUs = "Contact Us"
+}
+
+class AppInfoView: UIView {
+    let stackView = UIStackView()
```

Comment:
i think the stack view is unnecessary here, the use of it here adds a lot of code, you have only label and bottom line, it is a little bit over kill to use stack view
stack view used to save constraints but in this case its only adds layout code for us
i think it's better without it

Reply:
Done!